### DISCUSSION TITLE
Where do you set "FANCONTROL_CORSAIRLINK_ERROR_NOTIFICATIONS_DISABLED" to 1

### DISCUSSION THREAD

**Discussion by ZetsumeiIkari** [EvanMulawski/FanControl.CorsairLink#103] [Q&A] [answered]
What file do you edit for this setting
`General: Presents an alert if (10) consecutive attempts to refresh devices have failed. Like existing alerts, this can be disabled by setting the FANCONTROL_CORSAIRLINK_ERROR_NOTIFICATIONS_DISABLED environment variable to 1 and restarting Fan Control.`

**Comment by EvanMulawski** [comment, marked as answer]
See #78 - they are Windows environment variables.

**Reply by ZetsumeiIkari** [reply]
Thanks
